src/lib/bootloader/systemdboot.rb

```diff
@@ -21,14 +22,16 @@ class SystemdBoot < BootloaderBase
 
     CMDLINE = "/etc/kernel/cmdline"
 
-    # @!attribute menu_timeout
+    # @!attribute timeout
     #   @return [Integer] menu timeout
-    attr_accessor :menu_timeout
+    attr_accessor :timeout
```

Comment:
Hmm, this is backward incompatible change and it is already used in Agama to be ready for Systemd boot...is possible to add alias to those methods? https://github.com/agama-project/agama/blob/d71d924405f61a83de2b7cb8c1dfd82913f4c6da/service/lib/agama/storage/bootloader.rb#L127

Reply:
I have added the alias. Thanks!